scrunch/datasets.py

```diff
@@ -1628,6 +1632,49 @@ def _reload_variables(self):
             self._vars = self._catalog.index.items()
 
 
+class MissingRules(dict):
+    """
+    Handles variables missing rules in a dict fashion.
+    del var['skipped']  --> deletes a missing rule
+    """
+
+    def __init__(self, resource, *args):
+        self.resource = resource
+        # remove the nested key: {'value': value} to mimic
+        # pythonic dict behaviour
+        data = {}
+        for k,v in args[0].items():
+            data[k] = v['value']
+        dict.__init__(self, data)
+
+    def __setitem__(self, key, value):
+        data = {}
+        for k, v in self.items():
+            # wrap value in a {'value': value} for crunch
+            data[k] = {'value': v}
+            if key == k:
+                data[k]['value'] = value
+        # send the json to the missing_rules endpoint
+        result = self.resource.session.put(
+            self.resource.fragments.missing_rules,
+            json.dumps({'rules': data}))
+        assert result.status_code == 204
+        super().__setitem__(key, value)
+
+    def __delitem__(self, key):
+        assert self[key]
+        data = {}
+        for k, v in self.items():
+            # wrap value in a {'value': value} for crunch
+            data[k] = {'value': v}
+        del data[key]
+        result = self.resource.session.put(
+            self.resource.fragments.missing_rules,
+            json.dumps({'rules': data}))
+        assert result.status_code == 204
+        super().__delitem__(key)
```

Comment:
I get `TypeError: super() takes at least 1 argument (0 given)` when trying this.